app/models/host/base.rb

```diff
@@ -209,6 +211,10 @@ def ==(comparison_object)
         comparison_object.id == id
     end
 
+    def set_hostgroup(title)
+      self.hostgroup = Hostgroup.find_by_title(title)
```

Comment:
Call `set_hostgroup_defaults` after, so that the attributes from the host group are applied